migration/sql-files/084-type-groups-and-child-types.sql

```diff
@@ -39,4 +39,8 @@ CREATE TABLE work_item_child_types (
     position integer DEFAULT 0 NOT NULL
 );
 
-CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
\ No newline at end of file
+CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
+
+-- Only allow one work item link type with the same name for the same space
+-- template in existence.
+CREATE UNIQUE INDEX work_item_link_types_name_idx ON work_item_link_types (name, space_template_id) WHERE deleted_at IS NULL;
```

Comment:
here too, as above
and a newline please :smile:

Reply:
This one needs duplicate checking, correct.

Reply:
@pranavgore09 I remembered that I did clean up all link types in the DB in the [`080-remove-unknown-link-types.sql`](https://github.com/fabric8-services/fabric8-wit/blob/master/migration/sql-files/080-remove-unknown-link-types.sql) migration see also #1730. I don't think we need to do that here again. What do you think?

Reply:
okay then, np